src/WebJobs.Script/StorageProvider/AzureStorageProvider.cs

```diff
@@ -65,7 +65,6 @@ public virtual BlobContainerClient GetBlobContainerClient()
             if (!TryGetBlobServiceClientFromConnection(out BlobServiceClient blobServiceClient, ConnectionStringNames.Storage))
             {
                 var ex = new InvalidOperationException($"Could not create BlobServiceClient to obtain the BlobContainerClient using Connection: {ConnectionStringNames.Storage}");
```

Comment:
I meant you removed one exception but another one will be still logged
![image](https://user-images.githubusercontent.com/2578359/117892652-f2d29b00-b26d-11eb-9b72-9b5195d87839.png)

@balag0, do we also need to remove another one?

Reply:
I don't completely understand the scenario to comment on the intent behind these logs being exposed to customers :) It may be that we think these error messages are useful to customers.

I think the bigger point is - It is okay for function apps to not have AzureWebJobsStorage app setting set. Emitting these error logs due to the lack of that appsetting will generate support cases. If we were to start emitting these logs for an app that is not seeing error messages currently we should document what this error means what action is expected from customers.

Reply:
https://github.com/Azure/azure-functions-core-tools/blob/dev/src/Azure.Functions.Cli/Diagnostics/ColoredConsoleLogger.cs#L91

Looks like the core tools splits up the exception into multiple lines. Removing this log statement here should remove it from the cli tools. Working on validating this.